File: nodejs/sinmple_example (1).js
const axios = require('axios');

const webhookId = "YOUR_WEBHOOK_ID";
const webhookCode = "YOUR_WEBHOOK_CODE";

const url = "https://psyll.com/env/webhook/" + webhookId;

const payload = {
    code: webhookCode,
    action: "BUY",
    quantity: "0.01",
    symbol: "BTCUSDT"
};

axios.post(url, payload)
    .then(response => {
        console.log("Response:");
        console.log(response.data);
    })
    .catch(error => {
        console.error("Error:", error);
    });

File: ruby/sinmple_example (1).rb
require 'httparty'
require 'json'

webhook_id = "YOUR_WEBHOOK_ID"
webhook_code = "YOUR_WEBHOOK_CODE"

url = "https://psyll.com/env/webhook/#{webhook_id}"

payload = {
  code: webhook_code,
  action: "BUY",
  quantity: "0.0034",
  symbol: "BTCUSDT"
}

response = HTTParty.post(url, body: payload)

puts "Status code: #{response.code}"
puts "Response:"
puts JSON.pretty_generate(response.parsed_response)